telco-ran/configuration/kube-compare-reference/metadata.yaml

```diff
@@ -321,6 +321,13 @@ parts:
       - name: console-disable
         allOrNoneOf:
           - path: cluster-tuning/console-disable/ConsoleOperatorDisable.yaml
+  - name: fetch-only
```

Comment:
Really nice use of existing cluster-compare functionality. This would be a nice feature to propose into cluster-compare: specifically the ability to add to metadata.yaml a set of group-version-kind-namespace-name CRs to pull in only for context (so these placeholder CRs aren't needed just to pull them in). cc: @lack .

Reply:
Yes!

Or even scan all templates for "lookup" calls, and ensure the API/kind/namespace/name for those get auto-added to the fetch step...  I'll add a C-C backlog item about that.

Reply:
Added: https://issues.redhat.com/browse/CNF-19536